src/test.js

```diff
@@ -0,0 +1,105 @@
+import fs from 'fs';
+import { all, promisify, reject, resolve } from 'bluebird';
+import {
+    either,
+    evolve,
+    filter,
+    inc,
+    startsWith,
+    tryCatch
+} from 'ramda';
+import { compileES6 } from './compiler';
+import { getLocaleStrings } from './i18n';
+import { emitError, emitInfo, emitSuccess } from './input';
+import { inspect } from './module';
+import { compileSources } from './run';
+import { createVM, runAndGetAlerts } from './vm';
+
+const readFile = promisify(fs.readFile);
+
+/**
+ * Compiles the app files and returns an instance of a V8 object.
+ *
+ * @return {Promise<Object => Promise>}
+ */
+async function compileApp() {
+    const { name } = await readFile('package.json', 'utf-8')
+        | JSON.parse;
+    const [source, modules] = await compileSources();
+    const strings = await getLocaleStrings();
+    return context =>
+        runAndGetAlerts({ name, source }, context, strings, modules);
+}
+
+/**
+ * Compiles the test source.
+ *
+ * @return {Promise}
+ */
+async function compileTest() {
+    const source = await (readFile('test/index.js', 'utf-8')
+        .catchThrow(new Error('no tests provided')))
+        | compileES6;
+    const localTestModules = inspect(source).modules
+        | filter(either(startsWith('./'), startsWith('../')));
```

Comment:
E no ruindows?

Reply:
:thinking:

Reply:
I have no idea how it works, but it does. You can see `./` across a lot of source files of `rung-cli` (with `startsWith`).

Reply:
Ah, wait! This won't try to find the module! It will check for `import` or `require` with the module! So, there is no path solving!